src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
I agree about setQuery. What do you think of?
`setQuery (text, selectText = false)`
Can do as object just not very clear how will it make more "clear what the boolean stands for"?